server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -2199,7 +2199,7 @@ public Volume migrateVolume(MigrateVolumeCmd cmd) {
         // OfflineVmwareMigration: check storage tags on disk(offering)s in comparison to destination storage pool
         // OfflineVmwareMigration: if no match return a proper error now
         DiskOfferingVO diskOffering = _diskOfferingDao.findById(vol.getDiskOfferingId());
-        if (diskOffering.equals(null)) {
+        if (diskOffering == null) {
```

Comment:
This is the only one that makes sense to change.

Reply:
thanks for the hint, I did not know about this other specific use case.